restapi/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM mcr.microsoft.com/dotnet/core/sdk:2.2 AS build-env
```

Comment:
This file is in the right place but will most likely stop working as we move things around.
But do not of the top of my head know how to fix it.